### ISSUE TITLE
Installation instructions update

### ISSUE THREAD

**Issue by feacluster** [joalla/discogs_client#49]
The installation instructions should be changed from:

$ pip install python3-discogs-client

to

$ pip3 install python3-discogs-client

On two systems ( centos 7 and raspbian ), I had to use pip3 . Using pip defaulted to Python 2.7 installation.

**Comment by VinylVault** [comment]
added PR for this change

**Comment by alifhughes** [comment]
Yea I guess it doesn't hurt to add it, but I would argue that you would usually do this inside a virtual environment, which would have your desired Python version. So for me inside a virtual env of python3 using `pip` would default to `Python 3.x`.

**Comment by JOJ0** [comment]
I agree that it doesn't hurt to add this / clarify. Also @feacluster you probably had also have to use `sudo pip3 install ....`right? Because otherwise you usually wouldn't be allowed to install to your system-wide site-packages dir.

**Comment by feacluster** [comment]
> I agree that it doesn't hurt to add this / clarify. Also @feacluster you probably even had to use `sudo pip3 install ....`, is that correct? Because otherwise you usually wouldn't be allowed to install to your system-wide site-packages dir.

I checked my history and I did not use "sudo". I think regular pip3 install defaults to installing it locally?

**Comment by JOJ0** [comment]
> > I agree that it doesn't hurt to add this / clarify. Also @feacluster you probably even had to use `sudo pip3 install ....`, is that correct? Because otherwise you usually wouldn't be allowed to install to your system-wide site-packages dir.
> 
> 
> 
> I checked my history and I did not use "sudo". I think regular pip3 install defaults to installing it locally?

locally? I dont understand. you can install stuff as user root (or use sudo if you are a regular user to get root permission), only then you are allowed to the local system-wide python site-packages dir.

where you root? or where you in a python venv?

**Comment by feacluster** [comment]
I was not root or in a python env. It installed them to here:

``
pi@court4:~/.local/lib/python3.7/site-packages $ pwd
/home/pi/.local/lib/python3.7/site-packages
pi@court4:~/.local/lib/python3.7/site-packages $ ls -lrt
total 16
drwxr-xr-x 6 pi pi 4096 Apr 29 21:48 dateutil
drwxr-xr-x 2 pi pi 4096 Apr 29 21:48 python_dateutil-2.8.1.dist-info
drwxr-xr-x 3 pi pi 4096 Apr 29 21:48 discogs_client
drwxr-xr-x 2 pi pi 4096 Apr 29 21:48 python3_discogs_client-2.3.11.dist-info
pi@court4:~/.local/lib/python3.7/site-packages $
``

**Comment by JOJ0** [comment]
Ah ok that's what you mean. Yeah I didn't have this method on my radar. Yes usually installing to the current user's .local directory is possible providing the --user option to pip, as far as I know and after some research, reading this PEP: https://www.python.org/dev/peps/pep-0370/
 I found out that actually you are right, it seems that just hitting pip3 install, the packages land in .local/python - this seems to be the default. Tested on Ubuntu 18.04.

first I though this might be a speciality on the Pi's OS but it's default.

Anyway, let's not get to hung up here. I think just stating pip3 instead of pip would be sufficient - pip3 is as far as I know also available in a virtual python env (both is possible there, using pip but also pip3 epxlicitely - it's just the same)
